spring-cloud-starter-single-step-batch-job/src/main/java/org/springframework/cloud/task/batch/autoconfigure/kafka/KafkaItemWriterProperties.java

```diff
@@ -27,8 +27,15 @@
 @ConfigurationProperties(prefix = "spring.batch.job.kafkaitemwriter")
 public class KafkaItemWriterProperties {
 
+	/**
+	 * The topic name from which the messages will be written.
+	 */
 	private String topic;
 
+	/**
+	 * Indicate if the items being passed to the writer are all to be sent as delete
+	 * events to the topic.  Defaults to false.
```

Comment:
Remove the double space after the period (make it a single space) and make "false" be code: {@code false}